climada/entity/exposures/litpop/litpop.py

```diff
@@ -488,6 +498,9 @@ def from_shape(cls, shape, total_value, res_arcsec=30, exponents=(1,1), value_un
             Defining power with which lit (nightlights) and pop (gpw) go into LitPop.
         value_unit : str
             Unit of exposure values. The default is USD.
+        region_id : int, optional
+            The numeric ISO 3166 region associated with the shape. If ``None``, it will
+            be determined automatically (at a slight computational cost).
```

Comment:
I think this wording is slightly confusing. I thought `region_id` would be an argument to select the shape automatically. But, as I understand from this docstring, it is just the `region_id` that will be assigned to all grid points within the shape? How do you handle shapes that go over multiple regions?

Reply:
Well, I can only tell you what is implemented at the moment. You can input a single shape into `from_shape`. This will call `_get_litpop_single_polygon` once, and in this function, the `region_id` is either set to the input value or determined automatically. See here: https://github.com/CLIMADA-project/climada_python/blob/509326234c49e7534e7a9a3d46e35425d70499f0/climada/entity/exposures/litpop/litpop.py#L784

I just added the option to pipe the `region_id` through `from_shape`.

> But, as I understand from this docstring, it is just the region_id that will be assigned to all grid points within the shape?

This is what `_get_litpop_single_polygon` has been doing and still does, if you pass `region_id` to it.

> How do you handle shapes that go over multiple regions?

You either let the `region_id` be set automatically, or you choose a single ID for the entire shape (which might actually be something you want?)

Reply:
I see. Then we just let it as is. I maybe would just modify a bit the docstring wording and make it a bit more verbose (at least for the exposed methods).

Reply:
Done, see https://github.com/CLIMADA-project/climada_python/pull/720/commits/bf5d8c9ab621fe1445bac9a5998cfaea05657655